DS13/code/modules/overmap/combat.dm

```diff
@@ -0,0 +1,497 @@
+/mob/living/carbon/canMobMousedown(atom/object, location, params)
+	. = ..()
+	if(!overmap_ship)
+		return
+	. = overmap_ship
+
+
+/obj/structure/overmap
+	var/weapons_ready = TRUE //Are we ready to fire? This is a straight cooldown
+	var/weapons_cooldown = 25 //Fire every second second. Addtimer is slow so watch out
+	var/datum/beam/current_beam = null //Our phaser beam, wow!
+	var/damage = 0 //How much damage do we do? This is +10'd assuming weapons are powered!
+	var/charging = FALSE //charging weapons? if so, we can't fire
+	var/admin_override = FALSE //Need to test? set this on and you can fire and move yourself
+	var/fire_mode = "phaser"
+	var/photons = 4 //How many torpedoes do we have?
+	var/list/weapon_sounds = list('DS13/sound/effects/weapons/phaser.ogg','DS13/sound/effects/weapons/phaser2.ogg','DS13/sound/effects/weapons/phaser3.ogg','DS13/sound/effects/weapons/phaser4.ogg')
+	var/obj/structure/overmap/tractor_target = null //Are we tractoring a target? This forces it to move towards us.
+	var/datum/beam/tractor_beam = null
+	var/hail_ready = TRUE //Hailing cooldown
+	var/torpedo_damage = 30 //30 damage for free
+	var/area/linked_area = null
+	var/destroyed = FALSE
+	var/damage_sector = "shields" //Tactical can change this! What system are we targeting?
```

Comment:
Move all of these comments to be in a basic explainer section above in a /* comment area so it can be explained in a better format.